packages/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -6,21 +6,50 @@ import type { CardState } from './Card.types';
  */
 const useStyles = makeStyles({
   root: theme => ({
-    // TODO Add default styles for the root element
+    display: 'flex',
+    flexDirection: 'column',
+    overflow: 'hidden',
+
+    boxShadow: theme.alias.shadow.shadow4,
+    color: theme.alias.color.neutral.neutralForeground1,
+    backgroundColor: theme.alias.color.neutral.neutralBackground1,
+
+    // Size: medium
+    // TODO: Validate if we should use a token instead + the unit of said token
+    // TODO: Explore alternate way of applying padding
```

Comment:
At the moment, we're applying padding on the Card component instead of each child component for ease of implementation reasons. However, this means the users will need to revert this padding if they so have such a specific need, so perhaps having a spacing token and reusing this across all Card* components is a better way to go.